src/credentials/CertificationDeclaration.h

```diff
@@ -63,23 +63,25 @@ struct CertificationElements
     bool DACOriginVIDandPIDPresent;
 };
 
+struct CertificationDeclarationContent
+{
+    uint16_t formatVersion;
+    uint16_t vendorId;
+    bool containsPID;
+    uint32_t deviceTypeId;
+    char certificateId[kCertificateIdLength + 1];
+    uint8_t securityLevel;
+    uint16_t securityInformation;
+    uint16_t versionNumber;
+    uint8_t certificationType;
+    uint16_t dacOriginVendorId;
+    uint16_t dacOriginProductId;
+    bool dacOriginVIDandPIDPresent;
```

Comment:
Suggest adding defaults constructors for each (e.g. set defaults = 0 on scalars, etc), and suggest moving `certificateId` followed the two bools at the end to help alignment padding loss.

Reply:
Addressed in b5f45aa